airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```diff
@@ -211,14 +198,16 @@
                       }
                     }
                   }
-                ]
+                ],
+                "order": 1
               },
               "part_size_mb": {
-                "title": "Block Size (MB) for Amazon S3 multipart upload",
+                "title": "dddBlock Size (MB) for Amazon S3 multipart upload (Optional)",
```

Comment:
I think you added some random 'd's to this description?

Reply:
Seems yes, it was attempt to recognize whether specs is reloaded.